ydb/core/driver_lib/run/factories.h

```diff
@@ -40,7 +41,7 @@ struct TModuleFactories {
     // Factory for Simple queue services implementation details
     std::shared_ptr<NSQS::IEventsWriterFactory> SqsEventsWriterFactory;
 
-    IActor*(*CreateTicketParser)(const NKikimrProto::TAuthConfig&);
+    IActor*(*CreateTicketParser)(const NKikimrProto::TAuthConfig&, const NKikimr::TCertificateAuthValues&);
```

Comment:
why CertificateAuthValues are not part of AuthConfig?

Reply:
TAuthConfig is described in proto file. TCertificateAuthValues structure contains values from different places of configuration. May be create one common structure which will contain authconfig and other values from different configuration places?

Reply:
I think it would be better to have all auth-related settings in one place...

Reply:
done